woocommerce/woocommerce
# Update Icon for TikTok for WooCommerce extension

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33111.

This PR updates the icon used for the TikTok for WooCommerce extension. In doing so, the previous file is deleted because it refers the "TikTok for Business" logo. This file has not previously been included in a release.

<img width="707" alt="Screen Shot 2022-05-19 at 5 46 15 pm" src="https://user-images.githubusercontent.com/9312929/169266617-d11c5250-9351-4b3e-8871-2563b1db9a76.png">

### How to test the changes in this Pull Request:

1. Create a fresh site.
2. Make sure `woocommerce_show_marketplace_suggestions` is set to `no` in `wp_options` table.
3. Navigate to HTTP://your-site/wp-admin/admin.php?page=wc-admin&task=marketing 
4. Confirm the TikTok plugin is listed under `GROW YOUR STORE`
5. See that the normal TikTok logo is displayed (not TikTok for Business).

### Other information:




### FOR PR REVIEWER ONLY:
